windows/Dockerfile.build

```diff
@@ -0,0 +1,18 @@
+FROM microsoft/windowsservercore
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop';"]
+
+RUN Install-WindowsFeature NET-Framework-45-Core
```

Comment:
Is the .Net framework necessary? I compiled a C source in a Windows container only by installing the visualcppbuildtools https://github.com/StefanScherer/win-getaddrinfo/blob/3afdfab3f0fb7b1223d69ead19c93dbe917afed8/Dockerfile.msbuild#L5-L7